libs/reports-list/src/lib/reports-list-item/reports-list-item.component.html

```diff
@@ -1,4 +1,11 @@
 <div class="report-item-wrapper">
   <div>{{ healthCareService }}</div>
   <div>{{ cityName }}</div>
+  <div *ngIf="false" class="hidden-till-coming-report-is-complete">
+    <button (click)=comingReport($event) mat-button mat-raised color="primary">
+      <mat-icon aria-hidden="בדרך" aria-label="בדרך לקבל חיסון">directions_car</mat-icon>
```

Comment:
Cont. what we've talked, the text is not clear enough.
I thought about adding a label before the table, something like "ידע את האחרים שאתה מגיע ע"י לחיצה על "בדרך"", WDYT?